packages/visual-editor/src/components/puck/HeroSection.tsx

```diff
@@ -0,0 +1,399 @@
+
+  hours: {
+    entityField: YextEntityField<HoursType>;
+    showHours: boolean;
+  };
+  primaryCta: {
+    entityField: YextEntityField<CTAProps>;
+    variant: CTAProps["variant"];
+    showCTA: boolean;
+  };
+  secondaryCta: {
+    entityField: YextEntityField<CTAProps>;
+    variant: CTAProps["variant"];
+    showCTA: boolean;
+  };
+  image: {
+    entityField: YextEntityField<ImageType>;
+  };
+  styles: {
+    backgroundColor?: BackgroundStyle;
+    imageOrientation: "left" | "right";
+  };
+}
+
+//Define fields Should be of fields<props in step1>
+const defaultFields: Fields<HeroSectionProps> = {
+  businessName: {
+    label: "Business Name",
+    type: "object",
+    objectFields: {
+      entityField: YextEntityFieldSelector({
+        label: "Name",
+        filter: {
+          types: ["type.string"],
+        },
+      }),
+      level: BasicSelector("Heading Level", headingOptions),
+    },
+  },
+  localGeoModifier: {
+    label: "Local GeoModifier",
+    type: "object",
+    objectFields: {
+      entityField: YextEntityFieldSelector({
+        label: "Name",
+        filter: {
+          types: ["type.string"],
+        },
+      }),
+      level: BasicSelector("Heading Level", headingOptions),
+    },
+  },
+  hours: {
+    label: "Hours",
+    type: "object",
+    objectFields: {
+      entityField: YextEntityFieldSelector({
+        label: "Hours Field",
+        filter: {
+          types: ["type.hours"],
+        },
+      }),
+      showHours: {
+        label: "Show Hours",
+        type: "radio",
+        options: [
+          { label: "Show", value: true },
+          { label: "Hide", value: false },
+        ],
+      },
+    },
+  },
+  primaryCta: {
+    label: "Primary CTA",
+    type: "object",
+    objectFields: {
+      entityField: YextEntityFieldSelector({
+        label: "Value",
+        filter: {
+          types: ["type.cta"],
+        },
+      }),
+      variant: BasicSelector("Button Variant", [
+        { label: "Primary", value: "primary" },
+        { label: "Secondary", value: "secondary" },
+        { label: "Link", value: "link" },
+      ]),
+      showCTA: {
+        label: "CTA",
+        type: "radio",
+        options: [
+          { label: "Show", value: true },
+          { label: "Hide", value: false },
+        ],
+      },
+    },
+  },
+  secondaryCta: {
+    label: "Secondary CTA",
+    type: "object",
+    objectFields: {
+      entityField: YextEntityFieldSelector({
+        label: "Value",
+        filter: {
+          types: ["type.cta"],
+        },
+      }),
+      variant: BasicSelector("Variant", [
+        { label: "Primary", value: "primary" },
+        { label: "Secondary", value: "secondary" },
+        { label: "Link", value: "link" },
+      ]),
+      showCTA: {
+        label: "CTA",
+        type: "radio",
+        options: [
+          { label: "Show", value: true },
+          { label: "Hide", value: false },
+        ],
+      },
+    },
+  },
+  image: {
```

Comment:
I think this is right but it's not actually specified in the doc. I left a comment for Aaron.